cli_test/cli_test.go

```diff
@@ -1011,7 +996,7 @@ func TestGaiaCLIEncode(t *testing.T) {
 
 	// Check that the transaction decodes as epxceted
 	var decodedTx auth.StdTx
-	require.Nil(t, cdc.UnmarshalBinaryLengthPrefixed(decodedBytes, &decodedTx))
+	require.Nil(t, cdc.UnmarshalBinaryBare(decodedBytes, &decodedTx))
```

Comment:
@alessio this is what I meant. The internals of the test suite may break, but not the tests themselves.

Reply:
Yes, I understand your point better now, thanks!
The question is: is `decodedBytes` byte-to-byte identical to the previous output? If and only if the answer is yes, then no ABI change was introduced. Else we got an ABI breakage here

Reply:
Not the same, before transaction was encoded using LengthPrefix.

Reply:
yes, and what was stated in the changelog!

Reply:
Sure, it was mentioned in the `State Machine Breaking` stanza. My whole point is that this is worth to be mentioned in the `Client Breaking` stanza as well :)